src/tests/BaGet.Tests/TestServerBuilder.cs

```diff
@@ -24,47 +26,70 @@ public class TestServerBuilder
         /// </summary>
         private TestServerBuilder()
         {
+            Configuration = new Dictionary<string, string>();
         }
 
         private ITestOutputHelper _helper;
         private LogLevel _minimumLevel = LogLevel.None;
 
+        /// <summary>
+        /// In Memory representation of Config Settings
+        /// </summary>
+        public Dictionary<string, string> Configuration { get; private set; }
+
         /// <summary>
         /// Xunit.ITestOutputHelper is used as Logging-Target (Microsoft.Extensions.Logging)
         /// </summary>
         /// <param name="helper"></param>
         /// <param name="minimumLevel"></param>
         /// <returns></returns>
-        public TestServerBuilder TraceToTestOutputHelper(ITestOutputHelper helper, LogLevel minimumLevel )
+        public TestServerBuilder TraceToTestOutputHelper(ITestOutputHelper helper, LogLevel minimumLevel)
         {
             _helper = helper ?? throw new ArgumentNullException(nameof(helper));
             _minimumLevel = minimumLevel;
             return this;
         }
 
         /// <summary>
-        /// Empty means: 
-        ///  - Sqlite Database with a new/empty file
-        ///  - StorageType FileSystem based on a empty/temporary directory
+        /// Not configured Test Server Builder
         /// </summary>
         /// <returns></returns>
-        public static TestServerBuilder Empty()
+        public static TestServerBuilder Create()
         {
             return new TestServerBuilder();
         }
 
-        public TestServer Build()
+        private readonly string DatabaseTypeKey = $"{nameof(BaGetOptions.Database)}:{nameof(DatabaseOptions.Type)}";
```

Comment:
>DatabaseTypeKey [](start = 32, length = 15)

Let's put all the fields together, above the the constructor. Also, it looks like these can be `const` instead of `readonly`.

Reply:
"const" not working because we use $"{}" expressions on the right side => compiler can't calculate/assign this!